src/server.rs

```diff
@@ -473,11 +490,43 @@ impl ServerState {
         return ResponseCode::OK;
     }
 
+    fn remove_player(&mut self, player_id: PlayerID, player_cookie: String) {
+        let _opt_v = self.player_map.remove(&player_cookie);
+        let _opt_v = self.players.remove(&player_id);
+    }
```

Comment:
You can remove `let _opt_v = ` here :)

Reply:
Also, if you don't think this'll be used in other places, I would just move this to `handle_disconnect` below.

Reply:
For `_opt_v`, I think I added that in to silence the compiler warning since `HashMap::remove()` returns `Option` but we don't really care in this case, and we decided against `let _ =` syntax. And using the `?` operator would just pass it up the call stack, no?

Reply:
> Also, if you don't think this'll be used in other places, I would just move this to handle_disconnect below.

I want to keep `remove_player` for now. Better to have it than not need it.